### PR TITLE
Add a Group selection dropdown list

### PR DESCRIPTION
**Pull request by belljohan3**

### CODE REVIEW

File: cms/src/components/content/EditContentForm.vue [hunk 1]
```diff
@@ -214,7 +232,12 @@ const hasNoPublishErrors = computed(() => {
         return hasParentImage.value;
     }
 
-    return hasOneContentField.value && hasParentImage.value && hasTag.value;
+    return (
+        hasOneContentField.value && hasParentImage.value && hasTag.value && hasNoDraftErrors.value
+    );
+});
+const hasNoDraftErrors = computed(() => {
+    return hasGroup.value;
```
Comment (laurensgroeneveld) [inline cms/src/components/content/EditContentForm.vue:240]: I would suggest renaming `hasNoDraftErrors` to `canSaveAsDraft`, and then using that as well to disable the "save as draft" button (like `canPublish` does for publishing).

File: cms/src/components/content/GroupSelector.vue [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+import {
+    Combobox,
+    ComboboxButton,
+    ComboboxInput,
+    ComboboxLabel,
+    ComboboxOption,
+    ComboboxOptions,
+} from "@headlessui/vue";
+import type { Group } from "@/types"; // Assuming you have a Group type defined
+import LTag from "./LTag.vue"; // Assuming you have a component for rendering groups
```
Comment (laurensgroeneveld) [inline cms/src/components/content/GroupSelector.vue:13]: Not sure where these comments come from? Copilot? 😄

Comment (belljohan3) [reply cms/src/components/content/GroupSelector.vue:13]: 😄😄

**Comment by ivanslabbert** [comment]
@belljohan3 could you rebase this branch to main so that it does not fall to far behind? We'll get to implement it when done with #178

**Pull request closed without merging**